azkaban-web-server/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
The json is used for downloading mocha and chai whole libraries, which includes complete mocha component, which bowerInstall seems not having.